src/synthesizer_grids/grid_io.py

```diff
@@ -535,6 +535,67 @@ def add_specific_ionising_lum(self, ions=("HI", "HeII"), limit=100):
 
         self._close_file()
 
+    def add_bolometric_luminosity(self):
```

Comment:
To stop from having to call this individually in all scripts you could place a call to it after the spectra are written in `write_grid_common`